llvm/docs/LangRef.rst

```diff
@@ -11156,6 +11156,9 @@ until it reaches the size of the destination type, ``ty2``.
 
 When zero extending from i1, the result will always be either 0 or 1.
 
+If the ``nneg`` flag is set, and the ``zext`` argument is negative, the result
```

Comment:
The description of `nneg` is very minimal, vs the more detailed description in the RFC and PR. Maybe add to "Overview" something like "The `nneg` (non-negative) flag, if present, specifies that the operand is non-negative. This property may be used by optimization passes to later convert the `zext` to a semantically equivalent `sext`."

Reply:
Done! (I dropped the "semantically equivalent" bit because it's a refinement, not an equivalence.)